recipes/pulseaudio/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+import shutil
+
+
+class LibnameConan(ConanFile):
+    name = "pulseaudio"
+    description = "PulseAudio is a sound system for POSIX OSes, meaning that it is a proxy for sound applications."
+    topics = ("conan", "pulseaudio", "sound")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://pulseaudio.org/"
+    license = "LGPL-2.1"
+
+    generators = 'pkg_config'
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_alsa": [True, False],
+        "with_glib": [True, False],
+        "with_fftw": [True, False],
+        "with_x11": [True, False],
+        "with_openssl": [True, False],
+        # FIXME: enable when #2147 is merged
+        # "with_dbus": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_alsa": True,
+        "with_glib": False,
+        "with_fftw": True,
+        "with_x11": True,
+        "with_openssl": True,
+        # FIXME: enable when #2147 is merged
+        # "with_dbus": False,
+    }
+
+    build_requires = "gettext/0.20.1", "libtool/2.4.6"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _autotools = None
+
+    def configure(self):
+        if self.settings.os != "Linux":
```

Comment:
Please delete `fPIC` if shared.
Can you also re-order the methods? (config_options -> configure -> requirements?)
Can you also use the same quotes everywhere? Double quotes, I guess, since you used these the most.